### ISSUE TITLE
Can't turn some `firewalld` options to `no`

### ISSUE THREAD

**Issue by strarsis** [ansible-collections/ansible.posix#259]
##### SUMMARY
Some `firewalld` options can't be turned to `no`, they are set to `yet` instead.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
`ansible.posix.firewalld`

##### ANSIBLE VERSION
````
ansible 2.9.10
  config file = None
  configured module search path = ['/home/build/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.10 (default, Jun  2 2021, 10:49:15) [GCC 9.4.0]
````

##### CONFIGURATION

```paste below
(empty)
```

##### OS / ENVIRONMENT
`CentOS 7.x`

##### STEPS TO REPRODUCE
Set `firewalld` options that are represented by `yes` or `no` to `no`.
```yaml
    # Disable `ICMP block inversion` option
    - name: Set firewall `ICMP block inversion` option
      ansible.posix.firewalld:
        zone:   public
        permanent: yes
        state:  enabled
        icmp_block_inversion: 'no'
      notify: firewalld reload

    # Disable `masquerade` option
    - name: Set firewall `masquerade` option
      ansible.posix.firewalld:
        zone:   public
        permanent: yes
        state:  enabled
        masquerade: 'no'
      notify: firewalld reload
```

##### EXPECTED RESULTS
`icmp-block-inversion` and `masquerade` should be `no` (e.g. when checking with `firewall-cmd --list-all`).

##### ACTUAL RESULTS
`icmp-block-inversion` and `masquerade` are both `yes`

Also a warning is logged:

```paste below
[WARNING]: The value False (type bool) in a string field was converted to
u'False' (type string). If this does not look like what you expect, quote the
entire value to ensure it does not change.
```
Even using a (quoted) `no` string doesn't work, as it is still interpreted and set as `yes` instead.

**Comment by saito-hideki** [comment]
Hello, thank you for reporting this issue.
Sorry if I missed something, but I think this issue seems to be fixed on PR #234. Therefore if it is possible, can you try to apply this PR?

**Comment by saito-hideki** [comment]
@strarsis
As I mentioned earlier, this issue seems to be fixed in #234. So I'm closing this issue, but you can feel free to reopen it if I misunderstood about your issue.
